src/libxtp/gwbse/gw.cc

```diff
@@ -210,29 +219,35 @@ Eigen::VectorXd GW::SolveQP(const Eigen::VectorXd& frequencies) const {
   Eigen::VectorXd frequencies_new = frequencies;
   Eigen::Array<bool, Eigen::Dynamic, 1> converged =
       Eigen::Array<bool, Eigen::Dynamic, 1>::Zero(_qptotal);
+  bool fallback = _opt.qp_solver_fallback && _opt.qp_solver != "grid";
 #pragma omp parallel for schedule(dynamic)
   for (Index gw_level = 0; gw_level < _qptotal; ++gw_level) {
-    double initial_f = frequencies[gw_level];
+    double frequency = frequencies[gw_level];
     double intercept = intercepts[gw_level];
-    boost::optional<double> newf;
+    QPFunc fqp(gw_level, *_sigma.get(), intercept);
+    boost::optional<double> newf = boost::none;
+    // Call solver of choice
     if (_opt.qp_solver == "fixedpoint") {
-      newf = SolveQP_FixedPoint(intercept, initial_f, gw_level);
```

Comment:
yeah I do not want the fixed point solver anymore.

Reply:
Okay!